region/maridia/outer.json

```diff
@@ -1103,7 +1099,8 @@
                 {"and": [
                 "canSwim",
                   {"canShineCharge": {
-                    "usedTiles": 31
+                    "usedTiles": 31,
+                    "shinesparkFrames": 150
```

Comment:
It looks more like 65*2, so 130.

Reply:
Well... we don't know what room will be in there. Which actually is a new issue I hadn't considered. I guess just putting a few extra safety frames should mostly be enough.

Reply:
I'm including some frames on the other side as well for safety.